alldogs.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adoptadog - Seattle's premiere dog adoption agency</title>
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <header id='page-header'>
+      <a href='#start' class='skiplink'>Skip to content</a>
+      <div id='identity'>
+        <img class='logo' src="images/adoptadog-logo.svg" alt="Adopt a Dog logo">
```

Comment:
When using quote marks in HTML it's best to use double quotes <div id="identity"> We will reserve single quotes for javascript. It'll matter later on.

Reply:
Didn't realize! I'll clean that up as I'm working on the files.